.github/workflows/release.yaml

```diff
@@ -25,6 +25,8 @@ jobs:
         run: |
             # Filter out local helm-gh-pages image so we don't delete it
             docker system prune -a -f --filter "label!=org.opencontainers.image.source=https://github.com/stefanprodan/alpine-base"
+            sudo rm -rf /usr/share/dotnet
+            sudo rm -rf "$AGENT_TOOLSDIRECTORY"
```

Comment:
can you add some comments for why these two directories should be removed?

Reply:
Added a comment. The workaround is suugested here - https://github.com/actions/runner-images/issues/2840#issuecomment-790492173.